src/Resources/config/listener.yml

```diff
@@ -69,6 +69,20 @@ services:
         tags:
             - { name: kernel.event_listener, event: kernel.exception, method: onKernelException, priority: 96 }
 
+    contao.listener.header_replay.backend_session:
+        class: Contao\CoreBundle\EventListener\HeaderReplay\BackendSessionListener
+        arguments:
+            - "%contao.security.disable_ip_check%"
+        tags:
+            - { name: kernel.event_listener, event: t42.header_replay, method: onReplay }
```

Comment:
as far as I remember Symfony events, the *correct* way to name the method would be according to the event name. So `t42.header_replay` would result in an `onHeaderReplay` method. Also, if you follow that convention, you do not need to name the method (it will be discovered automatically).

On a side note, I think the event should be called `terminal42.header_replay` to not introduce another "namespace".